Tensile/KernelWriterSource.py

```diff
@@ -340,6 +340,8 @@ def functionPrefix(self, kernel):
     for i in range(1, kernel["ProblemType"]["NumIndicesC"]):
       indexChar = self.indexChars[i]
       kStr += " + (IDX%s)*strideD%s" % (indexChar, indexChar)
+    if kernel["_GlobalAccumulation"] == 'MultipleBuffer':
+      kStr += " + (gsuSumIdx)*strideW"
```

Comment:
strideW is not defined in GLOBAL_D macro.

Reply:
same design with other strideXXX